debian-minimal/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM        debian:jessie
+FROM        debian:buster
```

Comment:
could you please remove this altogether? there are no images based on this one, we can safely remove it (after merging, we'll also need to remove the image from dockerhub)

Reply:
Gotcha